woocommerce/sensei-course-participants
# Remove h3 heading from participant name

Fixes https://github.com/woocommerce/sensei-course-participants/issues/19

Note: command to build the CSS file is: `sass -s compressed --no-source-map assets/css/frontend.scss assets/css/frontend.css`

## Testing Instructions

Add the Course Participants widget to the sidebar. Ensure that everything behaves as before, except the user display names are not `h3` headings.
